.github/merge_rules.json

```diff
@@ -252,7 +147,8 @@
       "approved_by": ["kulinseth", "razarmehr"],
       "mandatory_checks_name": [
          "Facebook CLA Check",
-         "Lint"
+         "Lint",
+         "pull"
```

Comment:
Why? `pull` does not run any jobs relevant to this rule, does it?

Reply:
1. mostly for consistency
2. shouldn't pull eventually run mps tests?